test/regress/cli/regress1/quantifiers/sygus-inst-ufnia-sat-t3_rw1505.smt2

```diff
@@ -15,7 +15,7 @@ Publications: "Towards Bit-Width-Independent Proofs in SMT Solvers " by A. Nieme
 (declare-fun intand (Int Int Int) Int)
 (declare-fun intor (Int Int Int) Int)
 (declare-fun intxor (Int Int Int) Int)
-(define-fun bitof ((k Int) (l Int) (a Int)) Int (mod (div a (p2 l)) 2))
+(define-fun bit ((k Int) (l Int) (a Int)) Int (mod (div a (p2 l)) 2))
```

Comment:
I don't think that this change should be made since this does not have anything to do with the bit kind.